client/courses.html

```diff
@@ -48,14 +48,14 @@
   </ol>
   <!-- Example row of columns -->
   <div class="row">
-    <div class="col-xs-6 col-md-6">
+    <div class="col-md-6 col-sm-12 col-xs-12">
```

Comment:
I'm not positive, but I think the `col-xs-12` may be redundant with the `col-sm-12` on these.